### PR TITLE
fix builds on appveyor 64bit

### PR DESCRIPTION
**Pull request by wilzbach**
> Can't run '\bin\link.exe', check PATH

I think that setting the [LINKCMD64](http://stackoverflow.com/questions/21651357/64-bit-executables-with-dmd) is all that is needed, but AppVeyor takes quite a bit to run, so I leave this open here until verified.

Will ping you then ;-)

### CODE REVIEW

**Comment by wilzbach** [comment]
I am know a step further, but still get

```
LINK : fatal error LNK1104: cannot open file 'LIBCMT.lib'
```

https://ci.appveyor.com/project/greenify/mir/build/1.0.96/job/bgqk81u2kmjq8gs1

**Comment by wilzbach** [comment]
I managed to get 64bit and ldc working - as AppVeyor is already pretty overburdened I only selected the two recent dmd versions. However if you wish we can extend the matrix.

dmd: https://ci.appveyor.com/project/greenify/mir/build/1.0.114 (passing)
ldc: https://ci.appveyor.com/project/greenify/mir/build/1.0.121 (assertion failure)

File: appveyor.yml [hunk 1]
```diff
@@ -1,30 +1,106 @@
...
+    arch: x64
+  - DC: ldc
+    DVersion: 0.17.0
+    arch: x86
 
 skip_tags: true
+branches:
+  only:
+    - master
+    - stable
```
Comment (wilzbach) [inline appveyor.yml:31]: hopefully avoids double building of PRs

**Comment by wilzbach** [comment]
Correction: ldc on 32bit is not compiling :/

**Comment by wilzbach** [comment]
> Correction: ldc on 32bit is not compiling :/

It seems like they deprecated win32 - at least there no win32 releases for 1.0.0

https://github.com/ldc-developers/ldc/releases

-> removed ldc 32bit for windows

**Comment by wilzbach** [comment]
Wohoo - it is slowly [passing](https://ci.appveyor.com/project/9il/mir/build/1.0.100)

**Comment by wilzbach** [comment]
> Ok then I will add allow_failures temporarily to the appveyor script :(

done.

File: appveyor.yml [hunk 2]
```diff
@@ -1,30 +1,105 @@
...
+    arch: x86
+  - DC: ldc
+    DVersion: '1.0.0-alpha1'
+    arch: x64
+  - DC: ldc
+    DVersion: 0.17.0
+    arch: x64
+
+matrix:
+  fast_finish: true
```
Comment (wilzbach) [inline appveyor.yml]: this will abort the build once there's one failure - pretty handy feature!

File: appveyor.yml [hunk 3]
```diff
@@ -1,30 +1,105 @@
-platform:
-  - x86
-  - x64
+platform: x64
```
Comment (wilzbach) [inline appveyor.yml:1]: is it ok to compile 32bit applications on a 64bit platform (like we do on Travis)?
Setup is a lot easier this way

Comment (9il) [inline appveyor.yml:1]: Yes

**Comment by wilzbach** [comment]
@9il - everything is passing now (well we ignore ldc on windows). Could you please merge? :)

**Pull request merged**